kubernetes/build.gradle

```diff
@@ -21,8 +22,19 @@ task createScalaDockerfile(type: Dockerfile) {
 task createRDockerfile(type: Dockerfile) {
   destFile = outputFileR
   arg "spark_version"
-  from 'spark-r:$spark_version'
-  runCommand "apk add libc-dev linux-headers g++ libxml2-dev"
+  if (VersionNumber.parse(sparkVersion) >= VersionNumber.parse("2.4.0")) {
+    from 'spark-r:$spark_version'
+  } else {
+    from 'spark:$spark_version'
+  }
+  if (VersionNumber.parse(sparkVersion) >= VersionNumber.parse("2.4.4")) {
```

Comment:
Shouldn't be there `2.4.5` on this line?

Reply:
Yes, good catch!